bips/obpp-03.mediawiki

```diff
@@ -71,10 +73,27 @@ When one of the following cases exist, a compliant implementation may create an
 ## Add a spend output to the transaction.
 ## Add a change output to the transaction containing the difference between the current set of inputs and the desired spend.
 # Repeat step 1 to create a second spend output and change output.
+# Repeat step 2 zero or more times.
 # Adjust the change outputs as necessary to pay the desired transaction fee.
 
 Clients which create intentional HITs must have the capability to form alternate form HITs, and must do so for a non-zero fraction of the transactions they create.
 
+===Rules===
+
+A HIT formed via the proceding procedure will adhere to the following conditions:
+
+# The number of unique inputs scripts must exceed the number of output scripts.
+# All output scripts must be unique.
+# At least one pair of outputs must be of equal value.
+## "Standard outputs" refers to the set of outputs with equal value
+## "Standard value" refers to the value of the standard outputs
+## "Change outputs" refers to all outputs which are not standard outputs
+# For a HIT containing n standard outputs, there must exist at least one possible way to organize the inputs and outputs into n sets, where all sets satisfy the following:
+## The set contains one or more inputs, exactly one standard output, and exactly one change output
+## An input or output that appears in one set must not appear in any other set
+## The sum of the inputs in the set minus the value of the change output is equal to the standard value with a tolerance equal to the transaction fee.
+## Change outputs with a value of zero (virtual change outputs) are permitted. The are defined for the purpose of testing whether or not a HIT adheres to this specification but are not present in the version of the transaction which is broadcast to the network.
+
```

Comment:
This section could either use some re-wording for clarification, or some examples to illustrate them.